internal/builders/utils/access_command.go

```diff
@@ -0,0 +1,28 @@
+package utils
+
+import (
+	"bytes"
+	"text/template"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// CreateAccessCommand templates an access command string,
+// evaluates data from Pod/ExecAccessTemplates Metadata field
```

Comment:
Here you suggest its the PodAccessTemplate or ExecAccessTemplate metadata field - but you are actually passing in the `pod.ObjectMeta`. Perhaps your comment here should be more generic?

Reply:
Updated comment to be more generic `pod.ObjectMeta`